src/Qir/microsoft-quantum-qir-runtime-sys/build.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use cmake::Config;
+use std::boxed::Box;
+use std::env;
+use std::error::Error;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    println!("cargo:rerun-if-env-changed=TARGET");
+    println!("cargo:rerun-if-changed=build.rs");
+
+    for (key, value) in env::vars() {
+        println!("{}: {}", key, value);
+    }
+    if cfg!(target_os = "windows") {
+        let path_to_runtime_src = "..\\Runtime";
+        compile_runtime_libraries(path_to_runtime_src)?;     
+    } else {
+        let path_to_runtime_src = "../Runtime";
+        compile_runtime_libraries(path_to_runtime_src)?;
+    }
```

Comment:
(Nit)
Consider deduplicating as much as possible, such that only the differing part is placed in `if .. else ..`. Something like this (pseudocode):
```suggestion
    let path_separator = (cfg!(target_os = "windows") ? "\\" : "/");  // Either \ or /
    let path_to_runtime_src = ".." + path_separator + "Runtime";   // "..\Runtime" or "../Runtime"
    compile_runtime_libraries(path_to_runtime_src)?;
```